### PR TITLE
notify helper

### PR DESCRIPTION
**Pull request by socieboy**
Create a notify helper.

### CODE REVIEW

**Comment by socieboy** [comment]
How can I fix the travis-ci checks?

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -542,13 +542,13 @@ function method_field($method)
...
-     * @param $notification
-     * @return mixed
+     * @param  $users
+     * @param  $notification
+     * @return void
      */
     function notify($users, $notification)
     {
-        return app(Notificantion::class)->send($users, $notification);
+        app(Notificantion::class)->send($users, $notification);
```
Comment (zaknes) [inline src/Illuminate/Foundation/helpers.php]: 'Notification' is spelled wrong here and where it is pulled in at the top of this file.

File: src/Illuminate/Foundation/helpers.php [hunk 2]
```diff
@@ -13,6 +13,7 @@
 use Illuminate\Database\Eloquent\Factory as EloquentFactory;
 use Illuminate\Contracts\Validation\Factory as ValidationFactory;
 use Illuminate\Contracts\Broadcasting\Factory as BroadcastFactory;
+use Illuminate\Contracts\Notifications\Dispatcher as Notificantion;
```
Comment (brunogaspar) [inline src/Illuminate/Foundation/helpers.php]: Typo here `Notificantion`

File: src/Illuminate/Foundation/helpers.php [hunk 3]
```diff
@@ -537,6 +538,20 @@ function method_field($method)
...
+    /**
+     * Send to users a notification instance.
+     *
+     * @param  $users
+     * @param  $notification
+     * @return void
+     */
+    function notify($users, $notification)
+    {
+        app(Notificantion::class)->send($users, $notification);
```
Comment (brunogaspar) [inline src/Illuminate/Foundation/helpers.php]: And here.

**Comment by browner12** [comment]
please add types to the docblocks as well

**Comment by socieboy** [comment]
thanks, I already fix it, but still doesn't pass... sorry to bother you but im new on this, and i really want to contribute!

**Comment by Townfolio** [comment]
@socieboy Travis-CI might be having image problems. https://twitter.com/taylorotwell/status/765965520260653056

**Comment by socieboy** [comment]
Oh thanks! @Townfolio I will wait so!

**Comment by taylorotwell** [comment]
Going to hold off on this as I think most notifications will be sent via the Notifiable trait. If significant demand rises for it though we can revisit no problem.

**Comment by socieboy** [comment]
it's just a wrapper for:
```
Notification::send($users, new InvoicePaid($invoice));

notify($users, new InvoicePaid($invoice));
```
A little bit more clean and for all those who does't like facades.

**Comment by socieboy** [comment]
I have been using this helper on my personal projects instead the facade without problems, also when some notifications required to send to multiple users. I think you should consider to merge it!

**Pull request closed without merging**